core/models/configuration.rb

```diff
@@ -9,6 +9,7 @@ class Configuration
   attr_reader :node_names
   attr_reader :path
   attr_reader :provider
+  attr_reader :configuration_id
```

Comment:
Please put the description of the attribute to the top according to the alphabetical sort.